monitoring/monitorlib/scd_automated_testing/scd_injection_observation_api.py

```diff
@@ -0,0 +1,43 @@
+import enum
```

Comment:
[The API](https://github.com/interuss/dss/blob/d90153e203dd917fb57a5ab269146095567a64ab/interfaces/automated-testing/scd/scd.yaml#L3) mentions injection but not observation; it seems like the file name should be `scd_injection_api.py`.

Reply:
Thank you, I have now renamed the file.